create_pr_quick_start_guide.rst

```diff
@@ -0,0 +1,244 @@
+
+.. code:: bash
+
+  git fetch upstream
+  git rebase upstream/main
+
+12. Create a branch for your changes:
+
+.. code:: bash
+
+  git checkout -b name_of_my_branch
+
+13. We recommend you start with writing integration tests if applicable.
+
+Note: If there are any difficulties with writing / running the tests or you are not sure if the case can be covered, feel free to skip this step.
+If needed, other contributors can help you with it later.
+
+Note: Some collections do not have integration tests.
+
+All integration tests are stored in ``tests/integration/targets`` subdirectories.
+Go to the subdirectory containing the name of module you are going to change.
+For example, if you are fixing the ``mysql_user`` module in the ``community.mysql`` collection,
+its tests are in ``tests/integration/targets/test_mysql_user/tasks``.
+
+The ``main.yml`` file holds test tasks and includes other test files.
+Look for a suitable test file to integrate your tests or create and include a dedicated test file.
+You can use one of the existing test files as a draft.
+
+When fixing a bug, write a task which reproduces the bug from the issue.
+
+Put the reported case in the tests, then run integration tests with the following command:
+
+.. code:: bash
+
+  ansible-test integration name_of_test_subdirectory --docker -v
+
+For example, if the tests files you changed are stored in ``tests/integration/targets/test_mysql_user/``, the command will be:
+
+.. code:: bash
+
+  ansible-test integration test_mysql_user --docker -v
+
+You can use the ``-vv`` or ``-vvv`` argument, if you need more detailed output.
+
+In the examples above, the default test image will be automatically downloaded and used to create and run a test container.
+Use the default test image for platform independent integration tests such as those for cloud modules.
+
+If you need to run the tests against a specific distribution, see the `list of supported container images <https://docs.ansible.com/ansible/latest/dev_guide/testing_integration.html#container-images>`_. In this case, the command can look like:
+
+.. code:: bash
+
+  ansible-test integration name_of_test_subdirectory --docker centos8 -v
+
+Note: If you are not sure whether you should use the default image for testing or a specific one, skip the entire step - the community will help you later.
+You can also try to use the collection repository's CI to figure out which containers are used.
+
+If the tests do not want to run, first, check you complete step 3 of this guide.
+
+If the tests ran successfully, there are usually two possible outcomes:
+a) If the bug has not appeared and the tests have passed successfully, ask the reporter to provide more details. The bug can be not a bug actually or can relate to a particular software version used or specifics of local environment configuration.
+
+b) The bug has appeared and the tests has failed as expected showing the reported symptoms.
```

Comment:
I'm not sure why, but the a), b) here isn't showing up well if you view the file in preview mode. Here and the a/b below as well. Possibly it just needs a newline before the a) in each?

Reply:
Should be fixed, thanks for pointing out!